src/client/automation/playback/drag/base.js

```diff
@@ -36,9 +36,13 @@ export default class DragAutomationBase {
 
         this.automationSettings = new AutomationSettings(this.speed);
 
-        this.endPoint  = null;
-        this.downEvent = browserUtils.isTouchDevice ? 'touchstart' : 'mousedown';
-        this.upEvent   = browserUtils.isTouchDevice ? 'touchend' : 'mouseup';
+        this.endPoint        = null;
+        this.downEvent       = browserUtils.isTouchDevice ? 'touchstart' : 'mousedown';
+        this.upEvent         = browserUtils.isTouchDevice ? 'touchend' : 'mouseup';
+        this.dragAndDropMode = false;
```

Comment:
may be combine options required for drag&drop mode in one objest, smth like `this.dragAndDropState = {element: null, ...}`